exercises/dominoes/dominoes.rb

```diff
@@ -0,0 +1,8 @@
+class Dominoes
+
+  def can_chain?(dominoes_list = [])
+    result = false
+    return true if dominoes_list.length == 0 
+    result
```

Comment:
Yes, you are correct of course. Will do that as I get into refactoring..

"Serena Sees Her Footprints on the Moon" is on sale now @ iTunes! 
https://itun.es/us/CnX6bb.l

It's the story of a girl who imagines a trip to the Moon, where she interacts with some of the objects left behind up there by the astronauts from the Apollo missions.

No iPad or iPhone? No problem!
Get the book here: https://gum.co/softcover

> On Aug 28, 2016, at 6:10 PM, Tute Costa <notifications@github.com> wrote:
> 
> In exercises/dominoes/dominoes.rb:
> 
> > @@ -0,0 +1,8 @@
> > +class Dominoes
> > +
> > +  def can_chain?(dominoes_list = [])
> > +    result = false
> > +    return true if dominoes_list.length == 0 
> > +    result
> Is this the same as returning dominoes_list.length == 0?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>